timemachine/fe/free_energy.py

```diff
@@ -77,16 +77,22 @@ class HREXParams:
         If given, number of neighbor states on either side of a given replica's initial state for which to compute
         potentials. This determines the maximum number of states that a replica can move from its initial state during
         a single HREX iteration. Otherwise, compute potentials for all (replica, state) pairs.
+
+    optimize_target_overlap: float or None
+        If given, optimize the lambda schedule out of the initial bisection phase to target a specific minimum overlap
+        between all adjacent windows. Must be in the interval (0.0, 1.0] if provided.
```

Comment:
Huh, come to think of it, `optimize_target_overlap = 1.0` would probably lead to an infinite loop?

... Ahh, no, since `greedily_optimize_protocol` defaults to `max_iterations=1000`.

Maybe
```suggestion
        between all adjacent windows. Must be in the interval (0.0, 1.0) if provided.
```
or
```suggestion
        between all adjacent windows. Must be in the interval [0.0, 1.0) if provided.
```